testapp/src/main/res/layout/item_list_miniapp.xml

```diff
@@ -102,9 +101,8 @@
                 android:alpha="0.6"
                 android:textColor="@android:color/black"
                 android:textSize="@dimen/text_medium_14"
-                android:text="@string/lb_version_id"
+                android:text="@{miniapp.version.versionTag}"
```

Comment:
I'm not sure about this. Do we still need to display it?

Reply:
Yeah we still need it. It's from spec [MINI-111](https://jira.rakuten-it.com/jira/browse/MINI-111) [MINI-557](https://jira.rakuten-it.com/jira/browse/MINI-557)

Reply:
I hope so. @corycaywood Could you double confirm this pls?

Reply:
I think that's right. The version tag should be user visible.